CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -908,7 +908,7 @@ local function getProductInfo()
 	end
 
 	if not success or not result then
-		print("PurchasePromptScript: getProductInfo failed because", result)
+		warn("PurchasePromptScript: getProductInfo failed because", result, "Make sure a valid ID was specified")
```

Comment:
The only problem is that the error message provided by GetProductInfo is not clear if an invalid productId is specified. The error message is "MarketplaceService:getProductInfo() failed because rawProductInfo was empty", which does not clearly explain the issue.